### PR TITLE
Endpoint for The icelandic football status table.

### PR DESCRIPTION
**Pull request by alexanderbsig**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,15 +44,14 @@
     "cheerio": "~0.12.4",
     "express": "~3.20.1",
     "file": "~0.2.1",
+    "iconv-lite": "^0.4.13",
     "isn2wgs": "0.0.2",
     "jquery": "^2.1.4",
     "mocha": "~1.15.1",
     "moment": "~2.1.0",
-    "xml2js": "~0.2.8",
-    "request": "~2.27.0",
```
Comment (koddsson) [inline package.json]: Why were these two removed?

File: endpoints/isbolti/index.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+var app = require('../../server');
+var request = require('request');
+cheerio = require('cheerio');
```
Comment (koddsson) [inline endpoints/isbolti/index.js]: Missing `var` here?

**Comment by koddsson** [comment]
LGTM other from those two comments! :dart:

File: package.json (hunk 1 above)
Comment (alexanderbsig) [inline package.json]: Those lines are in my repo, I did not remove them, I cant see why they are red.

File: endpoints/isbolti/index.js (hunk 2 above)
Comment (alexanderbsig) [inline endpoints/isbolti/index.js]: Fixed :)

File: package.json (hunk 1 above)
Comment (MiniGod) [inline package.json]: Those lines appear to be duplicated in apis-is' master. This removal is legit

Comment (koddsson) [inline package.json]: Ah ok! :ok_hand:

**Comment by koddsson** [comment]
This looks good to me, it seems that @MiniGod has taken a peek at this :eyes: and the tests are passing so I'm merging this!

**Comment by koddsson** [comment]
Thanks for this @alexanderbsig! :tada:

**Comment by alexanderbsig** [comment]
:cake: for me  :smile:  
When will it be live?
Do I add the documentation for apis.is myself?

**Comment by kristjanmik** [comment]
It auto deploys when all tests are passing. However some data endpoints are broken and need to be fixed.

**Comment by alexanderbsig** [comment]
Allright sweet :smile:  what about the documentation for apis.is?

**Comment by kristjanmik** [comment]
@benediktvaldez can help you out there

**Comment by benediktvaldez** [comment]
Awesome stuff! Not sure how to proceed myself, since this is v1 stuff, and as stated in apis-is/docs#6 I have no idea where that is located.. 

@alexanderbsig I'll let you know when I know more! :rabbit:

**Comment by kristjanmik** [comment]
TBH it is a bit strange setup since we did not go through with the beta thing. The current "live" docs are located in this repo on the [gh-pages branch](https://github.com/apis-is/apis/tree/gh-pages)

**Pull request merged**